tests/integrations/server/lock_manager.rs

```diff
@@ -30,7 +30,12 @@ fn acquire_pessimistic_lock(
     req.for_update_ts = ts;
     req.lock_ttl = 20;
     req.is_first_lock = false;
-    client.kv_pessimistic_lock(&req).unwrap()
+    let mut resp = client.kv_pessimistic_lock(&req).unwrap();
+    if resp.get_region_error().has_stale_command() {
```

Comment:
Shouldn't be included in this PR? Maybe https://github.com/tikv/tikv/pull/6155 solves it.

Reply:
Yep. I think https://github.com/tikv/tikv/pull/6155 is ready for review.

Reply:
Ok, I have removed this code.